corehq/apps/analytics/README.md

```diff
@@ -0,0 +1,17 @@
+## Analytics
+We use multiple Analytics platforms. Google Analytics and Kissmetrics calls are done mainly using respective Javascript APIs. For Hubspot we use their REST API from server side as well.
+
+[Directory of events and properties we track in these platforms](https://docs.google.com/spreadsheets/d/1frMdFeznNcMAIyMW3pG3zes6mmY03UG67HyMUHXlb-s/edit#gid=1804103672)
+
+### Google Analytics and Kissmetrics
+See [analytics_all.html](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/style/templates/style/includes/analytics_all.html) and see example usages
+
+### Hubspot
+
+#### Adding/Removing Hubspot properties
+
+We track various user properties as [Hubspot Contact Properties](http://knowledge.hubspot.com/contacts-user-guide-v2/how-to-use-contact-and-company-properties). We have bunch of celery tasks and function that update these contact properties via [Hubspot REST API](http://developers.hubspot.com/docs/methods/contacts/create_or_update). We can update only those properties that are available on Hubspot portal. When we mention new Hubspot properties in HQ-Hubspot API calls, make sure that these properties are available in Hubspot portal. If they are not available, [create](http://knowledge.hubspot.com/contacts-user-guide-v2/how-to-create-contact-and-company-properties) them or ping marketing team to have them created on Hubspot portal first.
```

Comment:
its probably worth linking to our confluence page about hubspot development as well, and maybe update it because some of that information conflicts with what is on this page.